docs/installation/linux.md

```diff
@@ -720,6 +672,83 @@ chmod a+x PowerShell-6.0.0-beta.8-x86_64.AppImage
 dpkg -r powershell_6.0.0-beta.8-1.ubuntu.16.04_amd64.deb
 ```
 
+## Binary Archives
+
+PowerShell binary `tar.gz` archives are provided for macOS and Linux platforms to enable advanced deployment scenarios.
+
+### Dependencies
+
+For Linux, PowerShell builds portable binaries for all Linux distributions.
+But .NET Core runtime requires different dependencies on different distributions,
+and hence PowerShell does the same.
+
+The following chart shows the .NET Core 2.0 dependencies on different Linux distributions that are officially supported.
+
+| OS                 | Dependencies |
+| ------------------ | ------------ |
+| Ubuntu 14.04       | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu52 |
+| Ubuntu 16.04       | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu55 |
+| Ubuntu 17.04       | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu57 |
+| Debian 8 (Jessie)  | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu52 |
+| Debian 9 (Stretch) | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.2, libicu57 |
+| CentOS 7 <br> Oracle Linux 7 <br> RHEL 7 <br> OpenSUSE 42.2 <br> Fedora 25 | libunwind, libcurl, openssl-libs, libicu |
+| Fedora 26          | libunwind, libcurl, openssl-libs, libicu, compat-openssl10 |
+
+In order to deploy PowerShell binaries on Linux distributions that are not officially supported,
+you would need to install the necessary dependencies for the target OS in separate steps.
+For example, our [Amazon Linux dockerfile][amazon-dockerfile] installs dependencies first,
+and then extracts the Linux `tar.gz` archive.
+
+[amazon-dockerfile]: https://github.com/PowerShell/PowerShell/blob/master/docker/community/amazonlinux/Dockerfile
+
+### Installation - Binary Archives
+
+#### Linux
+
+```sh
+# Download the powershell '.tar.gz' archive
+curl -L -o /tmp/powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.9/powershell-6.0.0-beta.9-linux-x64.tar.gz
```

Comment:
should we include a step on how to install dependencies

Reply:
The steps to install dependencies is different for different distros and the dependencies are also different for different distros. Especially for unsupported distros, it would be an trail and error process (that's why this is for advanced deployment scenarios). I mentioned the Amazon Linux dockerfile as an example.

Reply:
I would suggest including one example and mentioning that it varies per distro.